neural-plasticity.csl

```diff
@@ -0,0 +1,145 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Neural Plasticity</title>
+    <title-short>Neural Plast</title-short>
+    <id>http://www.zotero.org/styles/neural-plasticity</id>
+    <link href="http://www.zotero.org/styles/neural-plasticity" rel="self"/>
+    <author>
```

Comment:
2 things:
- if you know/remember, please could you had the rel="template" link to the style you used to make this style.
- Could you add the link to the documentation? rel="documentation"